.gitignore

```diff
@@ -6,3 +6,4 @@ build
 .project
 .settings
 /work/
+bin
```

Comment:
Eclipse auto-generates that folder with all compiled sources and resources when imported and I don't want to push that or for it to pollute my uncommitted changes. Happy to remove it if you want though, since I can always just delete the folder :)